lib/transport_web/templates/atom/index.xml.eex

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<feed xmlns="http://www.w3.org/2005/Atom">
+
+  <title>Jeux de données GTFS transport.data.gouv.fr</title>
+  <subtitle>A subtitle.</subtitle>
+  <link href="<%= atom_url(@conn, :index) %>" rel="self" />
+  <id>tag:transport.data.gouv.fr,2019-02-27:/20190227161047181</id>
```

Comment:
is this normal that it's a fixed date?

Reply:
yes, the id is supposed to be constant as long as it represents the same thing.

To be honnest, I didn’t understand most of those things…